CHANGELOG

```diff
@@ -1,3 +1,105 @@
+cocos2d-x-3.17 May 2018
```

Comment:
Should change to exactly day before releasing.

Reply:
yep, I will do it